.changeset/soft-wombats-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+This change disallows creating standard refund when all items have been already refunded and disallows creating manual refund when user has no permissions to do so.
```

Comment:
**Suggestion:** I would remove "This change..." and replace it with "**Refunds:** Creating refund is disallowed when..." because when we will put all these logs in a list it will be difficult to read all of them. The other refund PR could get the same treatment :)